CMakeLists.txt

```diff
@@ -265,6 +285,7 @@ if(NETWORKIT_BUILD_CORE AND NETWORKIT_MONOLITH)
 	endif()
 
 	target_link_libraries(networkit PRIVATE OpenMP::OpenMP_CXX PUBLIC tlx)
+	target_link_libraries(networkit PRIVATE dhb)
```

Comment:
This again makes `dhb` mandatory. What is the point of using an option then?

Reply:
So the question here is rather: do we want to use `if` branches for example to remove the mandatory linkage to `dhb` or is there another way?

Reply:
Ok, I revoke my wrapper assumption. Maybe I again misunderstood the concept of integrating the library. `DHBGraph.hpp` seems to include two cases: one implementation where `dhb` is available and one where it is not? 

If so, what is the functionality of `DHBGraph.hpp`/`DHBGraph.cpp` if the library is not present? Seems very similar to graph then, right?